
---
float32 random_x
float32 random_y
